admin-ui/src/lib/admin-token.tsx

```
@@ -4,18 +4,33 @@ const ADMIN_TOKEN_STORAGE_KEY = "adminToken"
 
 export function readAdminToken(): string {
   if (typeof window === "undefined") return ""
-  return window.sessionStorage.getItem(ADMIN_TOKEN_STORAGE_KEY) || ""
+
+  const local = window.localStorage.getItem(ADMIN_TOKEN_STORAGE_KEY) || ""
```

ESLint

[error] Prefer `globalThis` over `window`.

(unicorn/prefer-global-this)